src/g3log/g3log.hpp

```diff
@@ -94,8 +94,13 @@ namespace g3 {
       bool isLoggingInitialized();
 
       // Save the created LogMessage to any existing sinks
+#ifdef ENABLE_WIN_WSTRING_SUPPPORT
+	  void saveMessage(const char *message, const wchar_t *wmessage, const char *file, int line, const char *function, const LEVELS &level,
```

Comment:
I understand and I had thought about it a long time. Code duplication was one issue, the other that if you decide to keep the concept of conversion in the sink, it is absolutely reasonable to add the stack trace and fatal message stuff as Utf-8 narrow strings (as most or all of it is plain ascii anyhow).